tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
Ah, I see. I wasn't aware of this for py3.4, so thanks for the clarification. 

Let's leave it as it is now and it's something I might have a mini spike/investigation later. I wouldn't want it to block all the other ace work.